lib/script.js

```diff
@@ -184,6 +184,8 @@ function wrapAsyncFunctions(asyncFunctions, sandbox, events, done, sandboxRoot)
         } else {
           args.push(function() {
             if (sandboxRoot._error) return;
+            // pairing of addCallback and finishCallback to keep callbackCount in right number
+            events.emit('addCallback');
```

Comment:
Adding this here for reference.

Add then immediately finish doesn't really seem like a good idea. Might as well not even finish it in the first place on the second line. This wouldn't have any effect this way.

Also, this change would break PR #484.